azkaban-web-server/src/test/java/azkaban/webapp/AzkabanWebServerTest.java

```diff
@@ -118,6 +121,22 @@ public void testInjection() throws Exception {
     final TriggerManager triggerManager1 = requireNonNull(injector.getInstance(TriggerManager.class));
     final TriggerManager triggerManager2 = requireNonNull(injector.getInstance(TriggerManager.class));
     assertTrue(triggerManager1 == triggerManager2);
+
+    //Test if TriggerLoader is singletonly guiced.
+    final TriggerLoader triggerLoader1= requireNonNull(injector.getInstance(TriggerLoader.class));
```

Comment:
That's a good suggestion. I am constructing a static assert singleton method in common Test module, such that everybody could call this.